src/net/Qml.Net/Internal/Types/NetReference.cs

```
@@ -25,8 +26,11 @@ public object Instance
         {
             get
             {
-                var handle = Interop.NetReference.GetHandle(Handle);
-                return ((GCHandle) handle).Target;
+                if(ObjectIdReferenceTracker.TryGetObjectFor(ObjectId, out var obj))
+                {
+                    return obj;
+                }
+                return null;
```

We should throw an exception detailing a fatal error here. This shouldn't happen, and if it does, many people will die.

We should probably output a message to stdout as well, considering .NET exceptions don't get passed along PInvoke boundaries.

Which brings me to another broader pointer of something we should eventually do (not now). At some point, we should wrap all callbacks with an exception handler, write the exception/stacktrace to stdout, and then throw the error.

This way, we can quickly tell the different between an actual native segfault issue (should be rare), and a .NET error on the reverse PInvoke side of things.

---

I created #37 for this.